app/views/content/train-to-be-a-teacher/qualifications-you-need-to-teach.md

```diff
@@ -1,10 +1,10 @@
 ---
 title: "What qualifications do I need to be a teacher?"
 heading: "What qualifications do I need to be a teacher?"
-subcategory: Qualifications and experience
+subcategory: Qualifications
 description: |-
   Find out what qualifications you need to be a teacher in primary and secondary schools in England and which subjects you can train to teach.
-navigation: 5.30
+navigation: 20.20
```

Comment:
This is the same number as in the GSE PR? They should all have a separate `navigation` number to set the order

Reply:
Having spoken to Martyn, we will resolve any conflicts like nav numbers once the first PR has been pushed live.